typescript/index.md

```diff
@@ -1,7 +1,16 @@
 # Module content
 
-- ToDo
-- ToDo
-- ToDo
-- ToDo
-- ToDo
+- Introduction​
+- - What is typescript?
+- - Why should I use typescript?
+- TypeScript features
+- - Base types
+- - Array
+- - Turples
+- - Functions
+- - Enum
+- - Type alias
+- - Unions
+- - Interface
+- - Generic
+- Usage in RN project
```

Comment:
Should we add an overview of setting typescript aliases?

Reply:
good point, agree with you